grpc/src/test/java/com/linecorp/armeria/server/grpc/UnframedGrpcServiceTest.java

```diff
@@ -97,7 +98,7 @@ void statusCancelled() throws Exception {
         final AggregatedHttpResponse res = response.aggregate().get();
         assertThat(res.status()).isEqualTo(HttpStatus.CLIENT_CLOSED_REQUEST);
         assertThat(res.contentUtf8())
-                .isEqualTo("grpc-code: CANCELLED, grpc error message");
+                .isEqualTo("{\"grpc-code\":\"CANCELLED\",\"message\":\"grpc error message\"}");
```

Comment:
c.f. As I removed [it](https://github.com/line/armeria/pull/3948/files#diff-2a4760cc7f5f469e4221418e09c2ae2a64d34baf4dcfd1c4447835c6ae612b0eL140-L141), `UnframedGrpcErrorHandler.ofJson` is used eventually.
So changed the response to JSON.